### ISSUE TITLE
OKFN version support for annotator 1.2.9?

### ISSUE THREAD

**Issue by cheekygit** [annotorious/annotorious-v1#103]
I want to use the filter functions available in annotator 1.2.9, but it appears to be incompatible with the OKFN version of Annotorious. The feather and instructions "Click and Drag to Annotate" display, but selecting an image area to annotate fails.

**Comment by rsimon** [comment]
Yes, sorry - due to lack of time I couldn't support the Annotator plugin as much as I would have liked to. Indeed the demo runs on Annotator v1.2.5, and I don't know when (and if...) I'll have the time to look into it.

Meanwhile though, the good folks over at @hypothesis have done lots of work on improving the pluign and making it work with their own fork of Annotator. I don't know the exact status of this work, but you may have more luck there. See:

https://github.com/gergely-ujvari/annotorious/tree/hypo-plugin

**Comment by rsimon** [comment]
P.S.: The most recent branch (cc-ing in @gergely-ujvari) seems to be 

https://github.com/gergely-ujvari/annotorious/tree/new-annotator-plugin

**Comment by gergely-ujvari** [comment]
I just want to add, that this branch is working with our fork of annotator, which is currently a quite different from the upstream.

Currently we're in the process or merging our improvements to upstream but for annotator 2.0 not annotator 1.2.x. 
Once that is complete, I'll port this plugin to the vanilla annotator 2.0. Currently it is a bit hard to use this plugin without our fork here: https://github.com/hypothesis/annotator/tree/typed-image-14 so I don't recommend using this right now, we'll clean this up and there'll be an official annotorious plugin for annotator 2.0
